service/assessment/assessment.go

```diff
@@ -375,29 +375,40 @@ func (svc *Service) handleEvidence(ev *evidence.Evidence, resourceId string) (er
 
 		log.Debugf("Evaluated evidence %v with metric '%v' as %v", ev.Id, metricID, data.Compliant)
 
-		targetValue := data.TargetValue
-
-		convertedTargetValue, err := convertTargetValue(targetValue)
+		// Get metric
+		metricConfiguration, err := svc.orchestratorClient.GetMetricConfiguration(context.Background(), &orchestrator.GetMetricConfigurationRequest{
```

Comment:
This should be unnecessary. The `data` field already contains the target value and operator that was actually used in the evaluation. The evaluation fetches the metric configuration from the orchestrator. 

The only thing that is missing from this is probably the cloud service ID (although this one is the same as in the evidence) and the "updated at". We could probably somehow loop the updated at also back to the `data` object.

Reply:
Here are the necessary lines to adjust: https://github.com/clouditor/clouditor/blob/0e86f5dff8ce965dfab9bb6d3c9b3ac818cdc9cb/policies/rego.go#L222-L224